.github/workflows/ci.yaml

```diff
@@ -84,7 +84,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: standard-v${{ matrix.clarity_version }}.wasm
-          path: ${{github.workspace}}/clar2wasm/src/standard/standard.wasm
+          path: ${{github.workspace}}/target/debug/build/clar2wasm-*/out/standard.wasm
```

Comment:
This doesn't seem to work : ```Error: No files were found with the provided path: /home/runner/work/clarity-wasm/clarity-wasm/target/debug/build/clar2wasm-*/out/standard.wasm. No artifacts will be uploaded.```
(source: https://github.com/stacks-network/clarity-wasm/actions/runs/14220378828/job/39849914693)